packages/quantic/force-app/main/default/lwc/quanticResultActionBar/quanticResultActionBar.js

```diff
@@ -0,0 +1,76 @@
+import {LightningElement} from 'lwc';
+
+/**
+ *  The `QuanticResultActionBar` component displays the Quantic Result Actions in a proper way with the right styles.
+ * @category Search
+ * @category Insight Panel
+ * @example
+ * <c-quantic-result-action-bar>
+ *   <c-quantic-result-action onclick={actionOne} icon-name="utility:attach" label-when-hover="Action One"></c-quantic-result-action>
+ *   <c-quantic-result-action onclick={actionTwo} icon-name="utility:email" label-when-hover="Action Two"></c-quantic-result-action>
+ *   <c-quantic-result-action onclick={actionThree} icon-name="utility:chat" label-when-hover="Action Three"></c-quantic-result-action>
+ * </c-quantic-result-action-bar>
+ */
+export default class QuanticResultActionBar extends LightningElement {
+  resultActionButtons = [];
+
+  connectedCallback() {
+    this.addEventListener('resultactionregister', this.handleButtonRegister);
+  }
+
+  disconnectedCallback() {
+    this.removeEventListener('resultactionregister', this.handleButtonRegister);
+  }
+
+  /**
+   * Handles the result action button registration event.
+   */
+  handleButtonRegister = (event) => {
+    event.stopPropagation();
+    const callbacks = event.detail;
+    const ref = event.target;
+    this.resultActionButtons.push({callbacks, ref});
+  };
+
+  /**
+   * Handles the slot change event.
```

Comment:
But what does it actually do? ;)

Reply:
I'll clarify that in the comment, it basically sends the order information to the QuanticResultAction components inside the slots.